### DISCUSSION TITLE
Get specific version of package for Apple Silicon

### DISCUSSION THREAD

**Discussion by 3nippo** [Homebrew/discussions#1061] [Everyday usage]
So I need to get a grpc bottle of version 1.17.2 for Apple Silicon. How do I do it properly? My suggestion is to make a user repository as explained [here](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/) with grpc@1.17.2 formula and then make a PR into main to obtain built bottles from BrewTestBot. But I don't think my bottles for grpc@1.17.2 are needed at main at all. Is there any other way to achieve what I want?

**Comment by carlocab** [comment]
Do you just need an M1-native version of 1.17.2 to install on your machine, or do you actually need the bottle to distribute to others?

**Reply by 3nippo** [reply]
No, I don't need the bottle to distribute it. I need the bottle because the project I recently joined stores 3rd party packages as unpacked bottles locally. 

Of course, there is an option to ignore this rule and try to build it from source, but what if the option with bottles is easier? And here I am, seeking for your help :)

**Reply by SMillerDev** [reply]
> No, I don't need the bottle to distribute it. I need the bottle because the project I recently joined stores 3rd party packages as unpacked bottles locally.

If someone uses homebrew infrastructure outside of homebrew we can't really provide support for that.

> Of course, there is an option to ignore this rule and try to build it from source, but what if the option with bottles is easier? And here I am, seeking for your help :)

Using bottles is really the only supported way to install software, and they will be used automatically if you run `brew install <something>`
